packages/ckbtc/src/utils/minter.utils.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { Cbor } from "@dfinity/agent";
+import { decodeMintMemo, LegacyMintMemoError } from "./minter.utils";
+
+describe("Transactions utils", () => {
+  it("should decode memo as a kyt fail", () => {
+    const kytFee = 1331;
+    const decodedMemo = [2, [kytFee, 1, 123]];
```

Comment:
What are 2, 1, and 123? Use constants?

Reply:
Copy/paste from NNS dapp.

Reply:
I guess it could have been clearer in NNS dapp as well, but there you can at least look at the code and see
```
    const mintType = decodedMemo[0];
    return mintType === MINT_TYPE_KYT_FAIL;
```
and realize that what matters its `decodedMemo[0] == MINT_TYPE_KYT_FAIL` (= 2).
Here, this is completely removed from that context.